src/corehost/cli/fxr/fx_ver.cpp

```diff
@@ -137,6 +249,10 @@ bool parse_internal(const pal::string_t& ver, fx_ver_t* fx_ver, bool parse_only_
     {
         return false;
     }
+    if (maj_sep > 1 && ver[maj_start] == _X('0'))
+    {
+        return false;
```

Comment:
Can you please add a comment for clarity - reading the C++ code is painful - so something like
``
// The value is a numeric which starts with padding zeroes - this is not allowed by the spec.
``

Reply:
Same below for minor and patch